src/App.jsx

```diff
@@ -1,25 +1,13 @@
 import React from 'react'
 import { GraphiQL } from 'graphiql'
 import 'graphiql/graphiql.css'
-import { useExplorerPlugin } from './grapfiql-explain/Explain'
-
-const fetcher = async gqlp => {
-  const data = await fetch('http://localhost:3001/graphql', {
-    method: 'POST',
-    headers: {
-      Accept: 'application/json',
-      'Content-Type': 'application/json'
-    },
-    body: JSON.stringify(gqlp),
-    credentials: 'same-origin'
-  })
-  return data.json().catch(() => data.text())
-}
+import { useExplainPlugin } from './grapfiql-explain'
```

Comment:
typo  `grapfiql` -> `graphiql`

Check please in other part of the project if there are similar errors.